reana_resources_k8s/templates/deployments/workers-template.yaml

```diff
@@ -24,10 +24,10 @@ spec:
         - name: EXPERIMENT
           value: {{worker.EXPERIMENT}}
         - name: SHARED_VOLUME
-          value: "/reana/{{worker.EXPERIMENT}}"
```

Comment:
Note that SHARED_VOLUME value is changed by this PR. Is it OK or should we adapt the "outside logic"?